stl/inc/xstring

```diff
@@ -1278,6 +1278,10 @@ public:
 #endif // _CONTAINER_DEBUG_LEVEL > 0
     }
 
+#if _HAS_CXX23
+    constexpr basic_string_view(nullptr_t) noexcept = delete;

```

Comment:
should we adopt the constructors above to add some `_STL_VERIFY`?

Reply:
> should we adopt the constructors above to add some `_STL_VERIFY`?

Hmm, but what will we verify though?

Reply:
That the passed pointer is not nullptr

Reply:
Just noticed this: if we keep `noexcept`, should I also add `// strengthened` at the end (unsure since it's a deleted constructor)?

Reply:
> That the passed pointer is not nullptr

Yeah I think that would be a good idea, but I also want to hear the maintainers' opinions.

Reply:
I believe the convention is to omit all qualifiers for a deleted special member function

Reply:
We generally don't check `some_pointer != nullptr` when we know the code will dereference `some_pointer` and segfault cleanly.